### DISCUSSION TITLE
Support Vim

### DISCUSSION THREAD

**Discussion by Koro-Reo** [EmranMR/tree-sitter-blade#74] [Q&A] [answered]
Hi! I found your project on a forum, but I noticed that's only for nvim and others text editors. So, my question is: maybe do you have in mind to support vim too? The other project was archived this year

**Comment by calebdw** [comment, marked as answer]
This is a general tree-sitter grammar that is (for the most part) editor agnostic. If Vim supports tree-sitter then yes this is supported, otherwise no.

**Reply by Koro-Reo** [reply]
I see. 

Well, I'll try later. Thx for the fast answer, I'm still kinda new using Vim

**Reply by calebdw** [reply]
> I'm still kinda new using Vim

So then come to the dark side and use Neovim instead---you have nothing to lose and everything to gain :sunglasses: 

Here's some good resources to get you started:
- https://github.com/nvim-lua/kickstart.nvim
- https://www.youtube.com/watch?v=m8C0Cq9Uv9o

**Reply by EmranMR** [reply]
If you ever got overwhelmed and wanted to cheat, I found https://nvchad.com/ a good option, having a nice default to start with 😅 (although my soul have been stolen by Helix).

You could always be brave and hit it raw though by using the links above 👀

**Reply by calebdw** [reply]
I don't particularly care for the Neovim **distributions**, now you have to learn how the distribution works and it just adds another layer of complexity when you're trying to customize things to your liking.

The kickstart.nvim repo above is not a distribution, it is designed to be a single file starting point that you can read (and understand what's going on) to get you up and going for customizing things yourself.

That said, if you're not the type to customize a bunch of things and just want something to work out of the box, then a distribution might be better for you.

**Reply by EmranMR** [reply]
That is the trap I fell into with the NvChad, at the end of the day you will eventually need to learn how to DIY.

**Reply by Koro-Reo** [reply]
Pls no the dark side!!! Honsetly I prefer stay with the classic, I spent a lot of time setting up everything hahaha but thx for this work
